addOns/custompayloads/src/main/resources/org/zaproxy/zap/extension/custompayloads/resources/Messages.properties

```
@@ -21,3 +21,8 @@ custompayloads.options.dialog.remove.text = Are you sure you want to remove this
 custompayloads.options.dialog.remove.button.confirm = Remove
 custompayloads.options.dialog.remove.button.cancel = Cancel
 custompayloads.options.dialog.remove.label = Do not show this message again
+
+custompayloads.options.dialog.addMultiplePayload.title = Multiple Payload
+custompayloads.options.dialog.addMultiplePayload.selectFile.button.name = SelectFile 
+custompayloads.options.dialog.addMultiplePayload.addPayload.button.name = AddMultiplePayload
+custompayloads.options.dialog.addMultiplePayload.duplicates.checkbox.label = CheckDuplicates
```

```suggestion
custompayloads.options.dialog.addMultiplePayload.selectFile.button.name = Select File 
custompayloads.options.dialog.addMultiplePayload.addPayload.button.name = Add Multiple Payloads
custompayloads.options.dialog.addMultiplePayload.duplicates.checkbox.label = Check Duplicates
```

The last one needs to be reworded and the "Allow Duplicates?" or "Prevent Duplicates" (and the logic tweaked if necessary)